test/system/dashboard/manage_internship_offers_test.rb

```diff
@@ -22,6 +22,7 @@ def fill_in_form(sector:, group:, weeks:, school_track:)
     if school_track
       select I18n.t("enum.school_tracks.#{school_track}"),
              from: 'internship_offer_school_track'
+      # internship_offer_type setup
```

Comment:
```suggestion
      # internship_offer_type setup
```
deadcode. git add -p